doc/devdocs/types.rst

```diff
@@ -319,9 +319,11 @@ the type, which is an object of type :obj:`TypeName`:
      primary: Array{T,N} <: DenseArray{T,N}
      cache: SimpleVector
        ...
+
      linearcache: SimpleVector
        ...
-     uid: Int64 47
+
+     uid: Int64 -7900426068641098781
```

Comment:
okay I get the same result - I bet this was changed by #17405, so it's intentional